<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmTrip</class>
 <widget class="QWidget" name="FrmTrip">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>668</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QLabel" name="lbHeader">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgb(247, 247, 247)</string>
       </property>
       <property name="text">
        <string>WRITE SOMETHING HERE LATER &lt;---</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbSource">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: qconicalgradient(cx:0, cy:0, angle:135, stop:0 rgba(220, 220, 220, 69), stop:0.375 rgba(255, 255, 0, 69), stop:0.423533 rgba(251, 255, 0, 145), stop:0.45 rgba(247, 255, 0, 208), stop:0.477581 rgba(255, 244, 71, 130), stop:0.518717 rgba(255, 218, 71, 130), stop:0.55 rgba(255, 255, 0, 255), stop:0.57754 rgba(255, 203, 0, 130), stop:0.625 rgba(255, 255, 0, 69), stop:1 rgba(255, 255, 0, 69));
font: 75 10pt &quot;Fixedsys&quot;;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="text">
        <string>[SOURCE]</string>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_23">
     <property name="text">
      <string>Choose Fishing Trip:</string>
     </property>
     <property name="buddy">
      <cstring>tableView</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_15">
     <item>
      <widget class="QTableView" name="tableView">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Choose cell (or create a new one)</string>
       </property>
       <property name="statusTip">
        <string>Choose cell (or create a new one)</string>
       </property>
       <property name="whatsThis">
        <string>In this widget we can choose cell, or create a new one</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QPushButton" name="pushNew">
         <property name="toolTip">
          <string>Create new record</string>
         </property>
         <property name="statusTip">
          <string>Create new record</string>
         </property>
         <property name="whatsThis">
          <string>Creates new record</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="app_new.qrc">
           <normaloff>:/app_new/filenew.png</normaloff>:/app_new/filenew.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QGroupBox" name="groupDetails">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Details</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Landing Site</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cmbSite"/>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Sampler</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cmbSampler"/>
        </item>
        <item>
         <widget class="QLabel" name="label_22">
          <property name="text">
           <string>Number of professional fishermen</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy">
           <cstring>spinProf</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinProf">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_24">
          <property name="text">
           <string>No part-time fishermen</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy">
           <cstring>spinPart</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinPart">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label_14">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Start</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="CustomTimeCtrl" name="customDtStart"/>
          </item>
          <item>
           <widget class="QLabel" name="label_25">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>End</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="CustomTimeCtrl" name="customDtEnd"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QGroupBox" name="groupTotals">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="title">
             <string>Totals</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="1">
              <widget class="QLabel" name="label_20">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Estimated</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2" colspan="2">
              <widget class="QLabel" name="label_21">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Calculated</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_16">
               <property name="text">
                <string>No of operations</string>
               </property>
               <property name="buddy">
                <cstring>spinNOE</cstring>
               </property>
              </widget>
             </item>
             <item row="2" column="4">
              <widget class="QLabel" name="label_15">
               <property name="text">
                <string>Units</string>
               </property>
               <property name="buddy">
                <cstring>cmbWeight</cstring>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_17">
               <property name="text">
                <string>Catch (weight)</string>
               </property>
               <property name="buddy">
                <cstring>spinCE</cstring>
               </property>
              </widget>
             </item>
             <item row="3" column="1" colspan="2">
              <widget class="QDoubleSpinBox" name="spinCE">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
             <item row="3" column="3">
              <widget class="QDoubleSpinBox" name="spinCC">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
             <item row="3" column="4" colspan="2">
              <widget class="QComboBox" name="cmbWeight"/>
             </item>
             <item row="4" column="4">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Weight/box</string>
               </property>
              </widget>
             </item>
             <item row="4" column="5">
              <widget class="QLabel" name="label_19">
               <property name="text">
                <string>Units</string>
               </property>
               <property name="buddy">
                <cstring>cmbBoxes</cstring>
               </property>
              </widget>
             </item>
             <item row="5" column="0">
              <widget class="QLabel" name="label_18">
               <property name="text">
                <string>Catch (no boxes)</string>
               </property>
               <property name="wordWrap">
                <bool>true</bool>
               </property>
               <property name="buddy">
                <cstring>spinCBE</cstring>
               </property>
              </widget>
             </item>
             <item row="5" column="1" colspan="2">
              <widget class="QDoubleSpinBox" name="spinCBE">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
             <item row="5" column="3">
              <widget class="QDoubleSpinBox" name="spinCBC">
               <property name="enabled">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="5" column="4">
              <widget class="QDoubleSpinBox" name="spinWeight">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
             <item row="5" column="5">
              <widget class="QComboBox" name="cmbBoxes"/>
             </item>
             <item row="1" column="3">
              <widget class="QSpinBox" name="spinNOC">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
             <item row="1" column="1" colspan="2">
              <widget class="QSpinBox" name="spinNOE">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_13">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Comments:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="textComments">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_12">
            <item>
             <widget class="QDialogButtonBox" name="buttonBox">
              <property name="standardButtons">
               <set>QDialogButtonBox::Apply|QDialogButtonBox::Close</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_16">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>548</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string/>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QPushButton" name="pushPrevious">
          <property name="text">
           <string>Previous</string>
          </property>
          <property name="icon">
           <iconset resource="app_new.qrc">
            <normaloff>:/app_new/back.png</normaloff>:/app_new/back.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushNext">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Next</string>
          </property>
          <property name="icon">
           <iconset resource="app_new.qrc">
            <normaloff>:/app_new/forward.png</normaloff>:/app_new/forward.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomTimeCtrl</class>
   <extends>QWidget</extends>
   <header>customtimectrl.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="app_new.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushNew</sender>
   <signal>clicked()</signal>
   <receiver>FrmTrip</receiver>
   <slot>createRecord()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>723</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>clicked(QAbstractButton*)</signal>
   <receiver>FrmTrip</receiver>
   <slot>onButtonClick(QAbstractButton*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>547</x>
     <y>505</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tableView</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>FrmTrip</receiver>
   <slot>previewRow(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>355</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>createRecord()</slot>
  <slot>onButtonClick(QAbstractButton*)</slot>
  <slot>previewRow(QModelIndex)</slot>
 </slots>
</ui>
